### ISSUE TITLE
Weird validation accuracy while using mxnet

### ISSUE THREAD

**Issue by uestcbingo** [apache/incubator-mxnet#1865]
I trained a residue-net like model recently using mxnet, however,  the validation error was kind of weird. Here are some details:

```
Epoch       Train-accuracy   Validation-accuracy
   3                  0.940604           0.694959
   4                  0.957220           0.529700
   5                  0.965952           0.657973
   6                  0.972351           0.721910
   7                  0.976373           0.695673
   8                  0.979318           0.316746
  ...                        ...                       ...
  12                 0.985547             0.719260
```

I thought this might be the consequence of overfitting, to figure it out, I replaced the validation set by training set (the full code is listed below ), here is part of the result.

```
Epoch            Train-accuracy         Validation-accuracy
   0                        0.148201                  0.468198    
   1                        0.749694                  0.663439
   2                        0.908540                  0.697312
   3                        0.945061                  0.714263
  ...                            ...                             ...
```

By the way ,this had never happened on other models(googlenet-like, or alexnet). I don't know if there's something wrong with my code or anything else.

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
